mesheryctl/pkg/utils/helpers.go

```diff
@@ -450,3 +450,26 @@ func ContentTypeIsHTML(resp *http.Response) bool {
 	}
 	return false
 }
+
+func SearchAndReplace(path, oldString, newString string) error {
+	fileData, err := ioutil.ReadFile(path)
+	if err != nil {
+		log.Errorf("unable to read file: %v", err)
+		return err
+	}
+
+	if !strings.Contains(string(fileData), oldString) {
+		log.Errorf("unable to replace with %s", newString)
+		return errors.New("unable to find requested string")
```

Comment:
Can we update this logic to consider for the case that someone may be "setting" or "switching" to the same channel that they are already subscribed to? In this case, we wouldn't error, but report success and a note that the setting remains unchanged.

As the return message is updated, can we move away from "string" and replace with "release channel"?

Reply:
I was thinking of handling this in the channel command, as I wanted to be the following function be a general utility not specific to channel command/

Reply:
Output from a test using the current logic in which the user is already subscribed to `stable`:

```
$ ./mesheryctl system channel --set stable
unable to replace with stable
Error switching channel:unable to find requested string
```